### PR TITLE
added support for setting background img size

### PR DESCRIPTION
**Pull request by AALMA**
this lets you specify the size of the image on the canvas, previously I was having issues where it was running off the canvas and in my case I needed to keep it within the canvas

### CODE REVIEW

File: jquery.jqscribble.js [hunk 1]
```diff
@@ -21,97 +21,116 @@ THE SOFTWARE.
...
+        backgroundImageY:       0,
+        backgroundImageWidth:   null,
+        backgroundImageHeight:  null,
+        backgroundColor:        "#ffffff",
+        saveMimeType:           "image/png",
+        saveFunction:           BasicCanvasSave,
+        brush:                  BasicBrush,
+        brushSize:              2,
+        brushColor:             "rgb(0,0,0)",
     fillOnClear: true
```
Comment (jimdoescode) [inline jquery.jqscribble.js]: Could you line this setting up with your other settings?

**Comment by jimdoescode** [comment]
This is great! Thank you! I made one minor style nit. Let me know if you can't get to it and I'll merge then fix it myself. Otherwise update the commit and I'll merge. 

Thanks again!

**Comment by AALMA** [comment]
Just committed a fix for that alignment issue, let me know if you need anything else.

**Comment by jimdoescode** [comment]
Merged. Thanks again!

**Pull request merged**